l10n_it_ricevute_bancarie/models/riba.py

```diff
@@ -423,8 +423,9 @@ def confirm(self):
                 to_be_reconciled |= riba_move_line.move_line_id
             move_line_model.with_context({"check_move_validity": False}).create(
                 {
-                    "name": "C/O %s-%s Ref. %s - %s"
+                    "name": "%s C/O %s-%s Ref. %s - %s"
```

Comment:
Tema per discussione: una stringa del genere non dovrebbe essere traducibile?

Ora, il contesto riba è inerentemente legato all'Italia, per cui la cosa ha importanza minore (lo lo considero bloccante per questa PR), però teoricamente uno potrebbe avere Odoo con l'interfaccia in cinese e usarlo per fare riba.